std/http/file_server.ts

```diff
@@ -126,7 +81,7 @@ async function serveFile(
   const fileInfo = await stat(filePath);
   const headers = new Headers();
   headers.set("content-length", fileInfo.len.toString());
-  headers.set("content-type", contentType(extname(filePath)) || "text/plain");
+  headers.set("content-type", "text/plain");
```

Comment:
Is this the behavior we want?

Reply:
If using `contentType()`, sometimes the browser will download the file directly

Reply:
@zhmushan Yeah but that was my intention... I remember Python `SimpleHTTPServer` similarly set content types

Reply:
I think we prefer to open a typescript file instead of downloading it

Reply:
> @zhmushan Yeah but that was my intention... I remember Python `SimpleHTTPServer/http.server` similarly set content types
> Ref: https://github.com/python/cpython/blob/305189ecdc8322c22879a04564cad5989f937462/Lib/http/server.py#L694, https://github.com/python/cpython/blob/305189ecdc8322c22879a04564cad5989f937462/Lib/http/server.py#L854-L876

It is a pity that the mime type of typescript has not been determined yet. Maybe we can do special processing for .ts files.

Reply:
Let's go with the behavior @zhmushan has changed for now - it's quite annoying when browsing files that they get downloaded...